### PR TITLE
(#1931710) rfkill compilation error fixes

### PR DESCRIPTION
**Pull request by msekletar**

### CODE REVIEW

**Review by dtardon — DISMISSED** [review]
LGTM

File: src/rfkill/rfkill.c [hunk 1]
```diff
@@ -336,7 +336,7 @@ static int run(int argc, char *argv[]) {
                 }
 
                 if ((size_t) l < RFKILL_EVENT_SIZE_V1)
-                        return log_error_errno(SYNTHETIC_ERRNO(EIO), "Short read of struct rfkill_event: (%zd < %d)",
+                        return log_error_errno(SYNTHETIC_ERRNO(EIO), "Short read of struct rfkill_event: (%zd < %ld)",
```
Comment (dtardon) [inline src/rfkill/rfkill.c:339]: Shouldn't this be %lu?

Comment (msekletar) [reply src/rfkill/rfkill.c:339]: Good point!

**Review by dtardon — APPROVED** [review]
LGTM

**Pull request merged**